recipes/font-ttf-ubuntu-feedstock/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = '0.83' %}
+
+package:
+  name: font-ttf-ubuntu
+  version: {{ version }}
+
+source:
+  url: https://assets.ubuntu.com/v1/0cef8205-ubuntu-font-family-{{ version }}.zip
+  sha256: 61a2b342526fd552f19fef438bb9211a8212de19ad96e32a1209c039f1d68ecf
+
+build:
+  number: 0
+  noarch: generic
+
+# test:
+#   commands:
+#     - test -f ${PREFIX}/fonts/Ubuntu-B.ttf       # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-BI.ttf      # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-C.ttf       # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-L.ttf       # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-LI.ttf      # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-M.ttf       # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-MI.ttf      # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-R.ttf       # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-RI.ttf      # [unix]
+#     - test -f ${PREFIX}/fonts/Ubuntu-Th.ttf      # [unix]
+#     - test -f ${PREFIX}/fonts/UbuntuMono-B.ttf   # [unix]
+#     - test -f ${PREFIX}/fonts/UbuntuMono-BI.ttf  # [unix]
+#     - test -f ${PREFIX}/fonts/UbuntuMono-R.ttf   # [unix]
+#     - test -f ${PREFIX}/fonts/UbuntuMono-RI.ttf  # [unix]
+
+about:
+  home: https://design.ubuntu.com/font/
+  summary: The Ubuntu Font Family
+  description: |
+    A unique, custom designed font that has a very distinctive look and feel.
+  license: Ubuntu Font Licence Version 1.0
+  license_file: LICENCE.txt
```

Comment:
This license doesn't seem to be the correct one for the font files.

Reply:
What do you mean? The archive of the url contains a file named copying.txt file stating `This Font Software is licensed under the Ubuntu Font Licence, Version 1.0.  https://launchpad.net/ubuntu-font-licence` and the same archive contains a file LICENCE.txt with that license.

Reply:
Ok, there is one thing. The zip archive behaves a bit different when extracted on linux/win vs. osx due to the included `__MACOSX/*` files. Those are ignored on extraction on osx, but not on linux/win, leading to one more subfolder level. The build.sh file handles that for all cases.

Reply:
The license file in the PR had a conda-forge copy right

Reply:
Ah, ok. The 4 font packages all reference the license file from their source archive in the recipe. The included LICENSE.txt files are for the two meta-packages and are just a copy of the staged-recipe/LICENSE.txt BSD license file with updated date. Is there any other preferred way to give licenses to meta-packages? May just `license: Various, license_family: Other` with no extra included `license_file`?